e2e/cli-e2e/project.json

```diff
@@ -23,7 +23,6 @@
     "utils",
     "core",
     "cli",
-    "cli",
     "plugin-eslint",
     "plugin-coverage",
     "plugin-js-packages",
```

Comment:
the new `../../global-setup.verdaccio.ts` will derive the dependencies. so you just need to add what you directly test. It's deps are installed automatically from the graph

Reply:
I wish that were the case, but if I omit indirect dependencies they're not published, it seems :disappointed: 

![image](https://github.com/user-attachments/assets/66bbc777-66ab-417f-90b7-2a513fc67e5a)

Reply:
Ah, nevermind, started working after I moved the working directory.

Reply:
I've decided to change `ci-e2e` `project.json` to limit the scope of this PR. There are redundant implicit dependencies all in the E2E projects, but I'd clean that up in some follow up PR.